lib/sequenceserver.rb

```diff
@@ -55,7 +55,7 @@ def run
       server = Thin::Server.new(config[:host],
                                 config[:port],
                                 :signals => false) do
-        use Rack::CommonLogger
+        use Rack::CommonLogger if SequenceServer.environment == 'development'
```

Comment:
1. Should make use of pre-defined convenience methods. See `SequenceServer.verbose?`.
2. Do not specify receiver unless required.